src/app/modules/main/chat_section/chat_content/users/controller.nim

```diff
@@ -80,8 +80,6 @@ proc init*(self: Controller) =
   self.events.on(SIGNAL_LOGGEDIN_USER_IMAGE_CHANGED) do(e: Args):
     self.delegate.loggedInUserImageChanged()
 
-  let chat = self.getChat()
-
```

Comment:
Was this just unused or how is this related?

Reply:
It was unused. I was looking for the place to edit the model so I was checking this controller and saw that this was useless. Just a clean up on the way.